catalogue/webapp/test/e2e/actions/search.ts

```diff
@@ -58,19 +60,28 @@ export async function clickActionFormatRadioCheckbox(
 export async function clickActionModalFilterButton(
   condition?: conditionSearchType
 ): Promise<void> {
-  const selector = `${
-    condition === 'images' ? searchImagesForm : searchWorksForm
-  } ${formatFilterMobileButton}`;
+  const searchImagesMobileButton = `${searchImagesForm} ${formatFilterMobileButton}`;
+  const searchCatalogueMobileButton = `${searchWorksForm} ${formatFilterMobileButton}`;
+
+  const selector =
+    condition === 'images'
+      ? searchImagesMobileButton
+      : searchCatalogueMobileButton;
 
   await page.click(selector);
 }
 
 export async function clickActionCloseModalFilterButton(
   condition?: conditionSearchType
 ): Promise<void> {
-  const selector = `${
-    condition === 'images' ? searchImagesForm : searchWorksForm
-  } ${mobileModalCloseButton}`;
+  const searchImagesCloseButton = `${searchImagesForm} ${mobileModalCloseButton}`;
```

Comment:
Should this just be stored in the selectors file?

Reply:
very good point .. moving it over

Reply:
when moving over these variables , i had issues with the selectors being undefined. Ill leave this for now and can work on this after.